### ISSUE TITLE
For a Lambda expression there is no inbound/outbound generated to link to method containing the lamda expression

### ISSUE THREAD

**Issue by mdpopa2001** [jeantessier/dependency-finder#6]
Hi,
I have the following sequence 
```
public class Test {
    public void someMethod(){
        Consumer<String> function = (x->System.out.println(x));
    }
}
```
and when generate the dependencies
 

```
<class confirmed="yes">
            <name>com.company.Test</name>
            <outbound type="class" confirmed="no">java.lang.Object</outbound>
            <feature confirmed="yes">
                <name>com.company.Test.Test()</name>
                <outbound type="feature" confirmed="no">java.lang.Object.Object()</outbound>
                <inbound type="feature" confirmed="yes">com.company.Main.main(java.lang.String[])</inbound>
            </feature>
            <feature confirmed="yes">
                <name>com.company.Test.lambda$someMethod$0(java.lang.String)</name>
                <outbound type="class" confirmed="no">java.io.PrintStream</outbound>
                <outbound type="feature" confirmed="no">java.io.PrintStream.println(java.lang.String)</outbound>
                <outbound type="class" confirmed="no">java.lang.String</outbound>
                <outbound type="feature" confirmed="no">java.lang.System.out</outbound>
            </feature>
            <feature confirmed="yes">
                <name>com.company.Test.someMethod()</name>
                <inbound type="feature" confirmed="yes">com.company.Main.main(java.lang.String[])</inbound>
            </feature>
        </class>
```

Problem:
There is no link(inbound/outbound) between the feature- com.company.Test.someMethod() and the feature com.company.Test.lambda$someMethod$0(java.lang.String).

Thank you

**Comment by jeantessier** [comment]
This is  because `CodeDependencyCollector` does not  know how to handle the JVM instruction `invokedynamic` (op code `0xBA`).  This is the instruction the JVM uses to call lambdas.  I have not updated language support to cover lambdas, yet.

**Comment by jeantessier** [comment]
The [initial spec for `invokedynamic`](https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-6.html#jvms-6.5.invokedynamic) is extremely complicated.  This will be a challenge.  On the bright side, the [latest spec](https://docs.oracle.com/javase/specs/jvms/se14/html/jvms-6.html#jvms-6.5.invokedynamic) is much more concise.  Fingers crossed!  🤞

**Comment by jeantessier** [comment]
Now that Dependency Finder is decompiling the `BootstrapMethods` attribute from `.class` files, I was able to follow the dependency from `someMethod` to the lambda in the example.  I need to do a little more testing and then I can make a release with this fix.

**Comment by jeantessier** [comment]
Fixed in `1.3.1`.
